pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -246,22 +246,29 @@ func (opt *Options) Run(suite *TestSuite) error {
 		includeSuccess = true
 	}
 
-	early, normal := splitTests(tests, func(t *testCase) bool {
+	early, others := splitTests(tests, func(t *testCase) bool {
 		return strings.Contains(t.name, "[Early]")
 	})
 
-	late, normal := splitTests(normal, func(t *testCase) bool {
+	late, others := splitTests(others, func(t *testCase) bool {
 		return strings.Contains(t.name, "[Late]")
 	})
 
+	kube, others := splitTests(others, func(t *testCase) bool {
```

Comment:
nit: can we use `kubeTests, openshiftTests` for future me?

Reply:
Done